environment.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
I would be in favour of just sorting alphabetically, again, for easily seeing differences across projects. Thoughts?